UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Services/FulfilmentDataService.cs

```diff
@@ -92,5 +106,18 @@ public async Task DownloadReadMeFile(string batchId, string exchangeSetRootPath,
                 logger.LogInformation(EventIds.DownloadReadMeFileRequestCompleted.ToEventId(), "Search and download ReadMe Text File completed for BatchId:{BatchId} and _X-Correlation-ID:{CorrelationId}", batchId, correlationId);
             }               
         }
+
+        public void CreateProductFile(string batchId, string exchangeSetInfoPath, string correlationId, SalesCatalogueDataResponse salesCatalogueDataResponse)
+        {
+            logger.LogInformation(EventIds.CreateProductFileRequestStart.ToEventId(), "Sales catalogue data response request started for product file for BatchId:{BatchId} and _X-Correlation-ID:{CorrelationId}", batchId, correlationId);
```

Comment:
Kindly change both log messages like "Create product file request started for BatchId

Reply:
As per the Comment i have change both log messages .